changelogs/fragments/11179-mas-list-parsing.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - mas - Parse CLI output correctly when listing installed apps (https://github.com/ansible-collections/community.general/pull/11179).
```

```suggestion
  - mas - parse CLI output correctly when listing installed apps (https://github.com/ansible-collections/community.general/pull/11179).
```
You also might to mention that this is because of a new version of `mas`.